tests/pipeline_runs_test.go

```diff
@@ -53,7 +59,32 @@ func (suite *IntegrationTestSuite) TestPipelineSuccessfulRun() {
 
 	suite.T().Run("Should create a Pipeline Run using custom pip server", func(t *testing.T) {
 		// Retrieve Pipeline ID to create a new run
-		pipelineDisplayName := "Test pipeline run with custom pip server"
+		pipelineDisplayName := "test-pipeline-run-with-custom-pip-server"
+		pipelineID, err := TestUtil.RetrievePipelineId(t, suite.Clientmgr.httpClient, APIServerURL, pipelineDisplayName)
+		require.NoError(t, err)
+		postUrl := fmt.Sprintf("%s/apis/v2beta1/runs", APIServerURL)
+		body := TestUtil.FormatRequestBody(t, pipelineID, pipelineDisplayName)
+		contentType := "application/json"
+		// Create a new run
+		response, err := suite.Clientmgr.httpClient.Post(postUrl, contentType, bytes.NewReader(body))
+		require.NoError(t, err)
+		responseData, err := io.ReadAll(response.Body)
+		responseString := string(responseData)
+		loggr.Info(responseString)
+		require.NoError(t, err)
+		require.Equal(t, 200, response.StatusCode)
+
+		err = TestUtil.WaitForPipelineRunCompletion(t, suite.Clientmgr.httpClient, APIServerURL)
+		require.NoError(t, err)
+	})
+
+	suite.T().Run("Should create a k8s Pipeline Run", func(t *testing.T) {
+		if suite.DSPA.Spec.APIServer.PipelineStorage != "kubernetes" {
```

Comment:
Why is this skipped?

Reply:
This test should be skipped when PipelineStorage is not set to kubernetes, because it uses a pipeline defined as a Kubernetes resource instead of uploading it through the API. If this test runs on a DSPA that uses a different PipelineStorage, it will fail.